### ISSUE TITLE
Contrast ratio in header not large enough for accessibility standards

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#518]
![Screen Shot 2023-01-11 at 14 57 35](https://user-images.githubusercontent.com/58090591/211905335-0c0d3f82-7053-47e5-a5f0-2b19d75daa85.png)

The grey text in the header on the NewCantus website _nearly_ meets WCAG AA/AAA accessibility standards.

The larger, main words ("Home", "About", etc.) technically pass the AA standard for large text, but it's not entirely clear that this text is large enough to be "large text", and otherwise fail across the board:
https://webaim.org/resources/contrastchecker/?fcolor=7F7F7F&bcolor=FFFFFF

The smaller descriptions ("Home Page", "Information") are a bit darker. If you sample a pixel in a dark area (like the cross of a "t" or an "f"), they pass the AA standard for normal text, but don't pass AAA:
https://webaim.org/resources/contrastchecker/?fcolor=616162&bcolor=FFFFFF

If you sample a lighter pixel (the colour below was chosen from an "a"), it fails both standards for normal text.
https://webaim.org/resources/contrastchecker/?fcolor=868686&bcolor=FFFFFF

Interacting with the homepage on a 10-year-old laptop, I found it difficult to make out the text in the headers unless I was looking at the screen precisely perpendicularly, with the screen brightness turned up quite high.

**Comment by jacobdgm** [comment]
in oldcantus, the items in the header are now black, rather than a pale grey! We should definitely change this, once the header/hamburger issue has been resolved.
